backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/handlingunits/rest_api/HandlingUnitsRestController.java

```diff
@@ -85,6 +87,8 @@ public class HandlingUnitsRestController
 {
 	public static final String HU_REST_CONTROLLER_PATH = MetasfreshRestAPIConstants.ENDPOINT_API_V2 + ENDPOINT_MATERIAL + "/handlingunits";
 
+	private static final String MORE_THAN_ONE_HU_FOUND_PARAM_NAME = "moreThanOneHUParamName";
```

Comment:
add javadoc, why this param is so important that it has it's own constant

Reply:
there is nothing special with it, just that I'm reading its value when handling the error, so I've considered it useful to have a constant to be able to easily see where the value is written and read